deepmd/.gitignore

```diff
@@ -0,0 +1,2 @@
+op/_*.py
```

Comment:
will this line also exclude `op/__init__.py`?

Reply:
That's true. I added another line to exclude it.